wheatley/row_generation/dixonoids_generator.py

```diff
@@ -68,7 +68,13 @@ def _gen_row(self, previous_row: Row, stroke: Stroke, index: int) -> Row:
             place_notation = self.plain_rules[0][pn_index]
 
         row = self.permute(previous_row, place_notation)
-        return row
+        return (row, None)
+
+    def summary_string(self) -> str:
+        """ Returns a short string summarising the RowGenerator.
+        This should make grammatical sense when formatted with 'Wheatley (will|is) ringing {summary_string}.'
+        """
+        return f"dixonoid: {self.plain_rules}"
```

Comment:
I can't find a more succinct way to summarise Dixonoids other than just printing their rules, but pretty printing would probably be useful here, particularly for the place notation.